Sources/Examples/Echo/Generated/echo.grpc.swift

```diff
@@ -212,7 +212,11 @@ class Echo_EchoServiceTestStub: ServiceClientTestStubBase, Echo_EchoService {
     return getResponses.first!
   }
   func get(_ request: Echo_EchoRequest, metadata customMetadata: Metadata, completion: @escaping (Echo_EchoResponse?, CallResult) -> Void) throws -> Echo_EchoGetCall {
-    fatalError("not implemented")
+    let response = try self.get(request)
+    let callResult = CallResult(success: true, statusCode: .ok, statusMessage: "OK", resultData: nil,initialMetadata: nil, trailingMetadata: nil)
+    completion(response,callResult)
+    let channel = Channel(address: "")
```

Comment:
Please inline `channel` with the line below. (also in the code generator)

Reply:
I guess you mean:
`return Echo_EchoGetCallBase( Channel(address: ""))`

If so, in my opinion this saves a line but it makes it much less readable. ( It reminds me to the "train wreck" properties problem )

Reply:
I don't know about that problem; but in that case please rename `channel` to `fakeChannel`.

Reply:
Sure, I will rename it and fix first for the rest of problems.

train wreck: http://wiki.c2.com/?TrainWreck